Kbuild.in

```diff
@@ -76,6 +76,7 @@ include $(src)/cm/repreb/Kbuild.sub
 include $(src)/dix/Kbuild.sub
```

Comment:
Test comment to check that this is allowed.

Reply:
test passed.